frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersionDeploy.cy.ts

```diff
@@ -260,100 +260,5 @@ describe('Deploy model version', () => {
         modelArtifactMocked.modelFormatVersion ?? ''
       }`,
     ).should('exist');
-
-    // Validate data connection section
-    cy.findByText(
-      "We've auto-switched to create a new data connection and pre-filled the details for you.",
-    ).should('exist');
-    kserveModal.findNewDataConnectionOption().should('be.checked');
-    kserveModal.findLocationNameInput().should('have.value', modelArtifactMocked.storageKey);
-    kserveModal.findLocationBucketInput().should('have.value', 'test-bucket');
-    kserveModal.findLocationRegionInput().should('have.value', 'test-region');
-    kserveModal.findLocationEndpointInput().should('have.value', 'test-endpoint');
-    kserveModal.findLocationPathInput().should('have.value', 'demo-models/test-path');
-  });
-
-  it('One match data connection on KServe modal', () => {
-    initIntercepts({});
-    cy.interceptK8sList(
-      SecretModel,
-      mockK8sResourceList([
-        mockSecretK8sResource({
-          namespace: 'kserve-project',
-          s3Bucket: 'dGVzdC1idWNrZXQ=',
-          endPoint: 'dGVzdC1lbmRwb2ludA==',
-          region: 'dGVzdC1yZWdpb24=',
-        }),
-        mockSecretK8sResource({
-          name: 'test-secret-not-match',
-          displayName: 'Test Secret Not Match',
-          namespace: 'kserve-project',
-          s3Bucket: 'dGVzdC1idWNrZXQ=',
-          endPoint: 'dGVzdC1lbmRwb2ludC1ub3QtbWF0Y2g=', // endpoint not match
-          region: 'dGVzdC1yZWdpb24=',
-        }),
-      ]),
-    );
-
-    cy.visit(`/modelRegistry/modelregistry-sample/registeredModels/1/versions`);
-    const modelVersionRow = modelRegistry.getModelVersionRow('test model version');
-    modelVersionRow.findKebabAction('Deploy').click();
-    modelVersionDeployModal.selectProjectByName('KServe project');
-
-    // Validate data connection section
-    kserveModal.findExistingDataConnectionOption().should('be.checked');
-    kserveModal.findExistingConnectionSelect().should('contain.text', 'Test SecretRecommended');
-    kserveModal.findLocationPathInput().should('have.value', 'demo-models/test-path');
-  });
-
-  it('More than one match data connections on KServe modal', () => {
```

Comment:
You're going to want to keep some test cases around but update them to use connection types